spec/features/reservations_spec.rb

```diff
@@ -836,4 +836,42 @@
       expect(page).to have_content('Description')
     end
   end
+
+  context 'when hide unavailable button is clicked' do
+    before(:each) do
+      empty_cart
+      Capybara.ignore_hidden_elements = false
+      @eq_model1 = FactoryGirl.create(:equipment_model, category: @category)
+      FactoryGirl.create(:equipment_item, equipment_model: @eq_model1)
+      @eq_model1[:overdue_count] = 1
```

Comment:
you should actually create an overdue reservation for this because it's a feature spec